erdpy/cli.py

```diff
@@ -199,44 +202,59 @@ def setup_parser_validators(subparsers):
     un_stake_parser = subparsers.add_parser("unstake")
     un_stake_parser.add_argument("--pem", required=True)
     un_stake_parser.add_argument("--value", default="0")
+    un_stake_parser.add_argument("--estimate-gas", action="store_true", default=False)
     un_stake_parser.add_argument("--gas-price", default=config.DEFAULT_GASPRICE)
-    un_stake_parser.add_argument("--gas-limit", default=config.DEFAULT_GASLIMIT)
+    un_stake_parser.add_argument("--gas-limit",
+                                 required=not ("--estimate-gas" in sys.argv),
```

Comment:
I see your point with using `sys.argv`. Could cause issues sometimes - see comment from this answer: https://stackoverflow.com/a/44211395/1475331. Anyway, should be okay for now.

Reply:
I think for now is ok :)